src/web-map.js

```diff
@@ -275,6 +276,36 @@ export class WebMap extends HTMLMapElement {
         this.dispatchEvent(new CustomEvent('createmap'));
       }
     }
+
+    /*
+      1. only deletes aria-label when the last (only remaining) map caption is removed
+      2. only deletes aria-label if the aria-label was defined by the map caption element itself
+    */
+
+    let mapcaption = this.querySelector('map-caption');
+    
+    if (mapcaption != null){
+      setTimeout(() => {
+        let ariaupdate = this.getAttribute('aria-label');
+  
+        if (ariaupdate == mapcaption.innerHTML) {
```

Comment:
Should be [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)